ringo_printtemplate/odfconv.py

```diff
@@ -67,6 +68,10 @@ def convert(self, data, format="ods", update=True):
         :returns: Converted data.
 
         """
+        # check document type before writing or opening
+        mimetype = check_mime_type_from_buffer(data)
+        if mimetype != "application/vnd.oasis.opendocument.text":
+            return None
```

Comment:
Okay, so I need a check in `_render_response` (did you mean build_response?), but not here.